cmd/cireport/main.go

```diff
@@ -47,8 +47,8 @@ func main() {
 			Client:  &client,
 		}
 
-		var realJobIDs string
```

Comment:
do we even need the variable `realJobIDs` anymore?

Reply:
We do, because jobIDs need to not changes between each jobs.
So realJobIDs is either the jobIDs if it was set, or the discovered job IDs otherwise.

Reply:
damn globals